### ISSUE TITLE
Can't install using Python 3.11

### ISSUE THREAD

**Issue by bonastreyair** [redis/redis-om-python#455]
pip basically does not found the latest versions when using python 3.11... (it only reaches until 0.0.27)

```
python3.10 -m pip index versions redis-om
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
redis-om (0.1.1)
Available versions: 0.1.1, 0.1.0, 0.0.27, 0.0.26, 0.0.25, 0.0.24, 0.0.23, 0.0.22, 0.0.21, 0.0.20, 0.0.19, 0.0.18, 0.0.17, 0.0.16, 0.0.15, 0.0.14, 0.0.13, 0.0.12, 0.0.11, 0.0.10, 0.0.9, 0.0.8, 0.0.7, 0.0.6, 0.0.5, 0.0.4, 0.0.3
```

```
python3.11 -m pip index versions redis-om
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
redis-om (0.0.27)
Available versions: 0.0.27, 0.0.26, 0.0.25, 0.0.24, 0.0.23, 0.0.22, 0.0.21, 0.0.20, 0.0.19, 0.0.18, 0.0.17, 0.0.16, 0.0.15, 0.0.14, 0.0.13, 0.0.12, 0.0.11, 0.0.10, 0.0.9, 0.0.8, 0.0.7, 0.0.6, 0.0.5, 0.0.4, 0.0.3
```

*Note: I found that all unit tests are running under python3.9.16*

![image](https://user-images.githubusercontent.com/37800834/210530767-2be9f44a-6f1d-4b04-b1d9-d29534af419d.png)

**Comment by YaraslauZhylko** [comment]
@dvora-h Is there a chance of releasing #446 anytime soon? Like `1.1.2` maintenance release?

**Comment by dvora-h** [comment]
I released version 0.1.2 with this fix
